### ISSUE TITLE
cant add cached_network_image 0.5.0 to pubspec.yaml

### ISSUE THREAD

**Issue by norbertkross** [renefloor/flutter_cached_network_image#92]
I tried adding 0.5.0 of the plugin to my pubspec.yaml file but I get the following
```
[fun_holidays] flutter packages get
Running "flutter packages get" in fun_holidays...               
Because every version of flutter_test from sdk depends on package_resolver 1.0.4 which depends on http ^0.11.0, every version of flutter_test from sdk requires http ^0.11.0.
And because cached_network_image >=0.5.0 depends on flutter_cache_manager ^0.2.0 which depends on http ^0.12.0, flutter_test from sdk is incompatible with cached_network_image >=0.5.0.
So, because fun_holidays depends on both cached_network_image ^0.5.0 and flutter_test any from sdk, version solving failed.

pub get failed (1)
```

**Comment by Adyel** [comment]
I have the same issue. Currently running 
```
[✓] Flutter (Channel beta, v0.9.4-pre.28, on Linux, locale en_US.UTF-8)
    • Flutter version 0.9.4-pre.28 at /home/adyel/opt/flutter
    • Framework revision dc14cefae8 (6 weeks ago), 2018-09-25 17:45:40 -0400
    • Engine revision 74625aed32
    • Dart version 2.1.0-dev.5.0.flutter-a2eb050044

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /home/adyel/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /var/lib/snapd/snap/android-studio/69/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] Android Studio (version 3.2)
    • Android Studio at /var/lib/snapd/snap/android-studio/69/android-studio
    • Flutter plugin version 30.0.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] Connected devices (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)
```

**Comment by renefloor** [comment]
Fixed, use 0.5.0+1

**Comment by kyungjoongo** [comment]
same issue 
cant add cached_network_image 0.5.1 to pubspec.yaml
